site/content/en/docs/manual/advanced/single-shape.md

```diff
@@ -0,0 +1,99 @@
+| **Drop-Down List for Label Selection** | Allows for the selection of different labels for annotation within the interface.                                                                                                                                                                                                                                                                            |
+| **Options to Enable or Disable**       | Provides configurable options to streamline the annotation process, such as: <br> - Automatically advancing to the next frame after drawing an object. <br> - Auto-saving upon completion of annotations on all frames. <br> - Navigating only through frames that haven't been annotated yet.                                                               |
+| **Predefined Number of Points**        | Specific to polyshape annotations, enabling this option auto-completes a shape once a predefined number of points is reached. Otherwise, pressing **N** is required to finalize the shape.                                                                                                                                                                   |
+| **Number of Points**                   | Applicable for polyshape annotations, indicating the number of points to utilize for image annotation.                                                                                                                                                                                                                                                       |
+
+<!--lint enable maximum-line-length-->
+
+## Annotating in Single Shape mode
+
+To annotate in Single Shape mode, follow these steps:
+
+1. Open the job and switch to **Single Shape** mode.
+2. Annotate the image based on the selected shape.
+   For more information on shapes, see [Annotation Tools](http://localhost:1313/docs/getting_started/overview/#annotation-tools).
+3. (Optional) If the image does not contain any objects to annotate,
+   click **Skip** at the top of the right panel.
+4. Submit your work.
+
+## Query parameters
+
+Also, we introduced additional query parameters, which you may append to
+the job link, to initialize the annotation process and automate workflow:
+
+<!--lint disable maximum-line-length-->
+
+| Query Parameter      | Possible Values                                                             | Explanation                                                                                          |
+| -------------------- | --------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| `defaultWorkspace`   | Workspace identifier (e.g., `single_shape`, `tags`, `review`, `attributes`) | Specifies the workspace to be used initially, streamlining the setup for different annotation tasks. |
```

Comment:
Shall we add this information to corresponding workspaces documentation?

Reply:
@bsekachev we should but I think as a separate task it will take some time to understand where exactly to add them. I will create a card for this one.